From b44f19d7038f7e6182db4d29aee39dab569193fd Mon Sep 17 00:00:00 2001
From: Colin Guthrie <cguthrie@mandriva.org>
Date: Sat, 24 Apr 2010 14:25:31 +0100
Subject: [PATCH] xine: Initialise volume to 1.

When PulseAudio is used, it's essential that the internal representation
of the volume remains at the maxium. Xine will use this to set the initial volume.

Thankfully, xine's PulseAudio output does not seem to use per-application volume control
as otherwise this 100% would be passed on an prevent PA from saving
volumes properly.

Currently, however one more problem remains with Xine+PA+volume
in that the standard volume widget does not seem to be hooked up
correctly. I'll look at this problem next.
BUG: 235193
---
 xine/audiooutput.cpp |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/xine/audiooutput.cpp b/xine/audiooutput.cpp
index 05b9240..f9f922e 100644
--- a/xine/audiooutput.cpp
+++ b/xine/audiooutput.cpp
@@ -48,6 +48,7 @@ namespace Xine
 
 AudioOutput::AudioOutput(QObject *parent)
     : AbstractAudioOutput(new AudioOutputXT, parent)
+    , m_volume(1.0)
 {
 }
 
-- 
1.6.1

